woocommerce/woocommerce
# Try using iframe for loading assembler hub behind the scenes

### Changes proposed in this Pull Request:

Attempt using iframe for temporary solution on https://github.com/woocommerce/woocommerce/issues/40793

Closes https://github.com/woocommerce/woocommerce/pull/40936

### How to test the changes in this Pull Request:

1. Make sure to enable `customize-store` feature flag
1. Go to `WooCommerce -> Home` and click `Customize Store` task
1. Go through the AI wizard and observe the assembler hub showing up relatively quickly after the loading screen
1. Please test around navigating links within the iframe, it should behave like it's the original window
2. Please test with other browsers
